devices/Ds18b20/README.md

```diff
@@ -0,0 +1,184 @@
+# Ds18b20 - Temperature Sensor
+
+The DS18B20 digital thermometer provides 9-bit to 12-bit Celsius temperature measurements and has an alarm function with nonvolatile user-programmable upper and lower trigger points.
+
+## Documentation
+
+Product datasheet can be found [here](https://datasheets.maximintegrated.com/en/ds/DS18B20.pdf)
+
+## Sensor Image
+
+![sensor](sensor.png)
+
+## Circuit
+
+![circuit](Ds18b20.png)
+
+* DATA - RX2 + TX2 shorted
+* VCC - 3.3V
+* GND - GND
+
+**Important**: make sure you've connected data pin of sensor to two pins on the board!
+
+## Usage
+
+**Important**: make sure you properly setup the Rx/Tx pins especially for ESP32 before creating the `OneWireHost`, make sure you install the `nanoFramework.Hardware.ESP32 nuget`:
+
+```csharp
+Configuration.SetPinFunction(16, DeviceFunction.COM3_RX);
+Configuration.SetPinFunction(17, DeviceFunction.COM3_TX);
+```
+
+### Library supports three approaches to read the data from Ds18b20 sensors:
+
+1. Reading from first found sensor on OneWire bus or sensor with address passed as second parameter to the constructor:
+
+```csharp
+private static void ReadingFromOneSensor()
+{
+    OneWireHost oneWire = new OneWireHost();
+
+    Ds18b20 ds18b20 = new Ds18b20(oneWire, null, false, TemperatureResolution.VeryHigh);
+
+    ds18b20.IsAlarmSearchCommandEnabled = false;
+    if (ds18b20.Initialize())
+    {
+        Console.WriteLine($"Is sensor parasite powered?:{ds18b20.IsParasitePowered}");
+        string devAddrStr = "";
+        foreach (var addrByte in ds18b20.Address)
+        {
+            devAddrStr += addrByte.ToString("X2");
+        }
+        Console.WriteLine($"Sensor address:{devAddrStr}");
+
+        while (true)
+        {
+            ds18b20.Read();
+
+            Console.WriteLine($"Temperature: {ds18b20.Temperature.DegreesCelsius.ToString("F")}\u00B0C");
```

Comment:
you should update the readme ;-)

Reply:
Done, I totally forgot about it, thanks!